plop-templates/Component/Component.tsx.hbs

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from "react";
+
+import classNames from "classnames/bind";
+import styles from "./{{pascalCase name}}.module.css";
+
+const cx = classNames.bind(styles);
```

Comment:
things be changing every day 😅  It seems we're all on board with camelcase classnames and replacing `classNames.bind` with `clsx` (and you wouldn't need to worry about paramCase!)

Reply:
😞 sorry for flip flopping - I think we're set now with camelCase + clsx (#362 option 3)

Reply:
Np! It's expected while these things are still being settled.